decidim-core/spec/lib/form_builder_spec.rb

```diff
@@ -95,6 +95,8 @@ def self.model_name
 
           expect(parsed.css("li.tabs-title a").count).to eq 3
 
+          expect(parsed.css(".editor label[for='resource_short_description_en']").first).to be_nil
```

Comment:
I don't think we should add this specific test here. Following Gary Bernhardt's DAS recommendation, use really tests to chase bugs, remove them afterwards. This introduces coupling that can end up biting us.